tools/format.py

```diff
@@ -17,31 +17,19 @@
 os.chdir(root_path)
 
 run([clang_format_path, "-i", "-style", "Google"] +
-    find_exts("libdeno", ".cc", ".h"))
+    find_exts("libdeno", [".cc", ".h"]))
 
-for fn in ["BUILD.gn", ".gn"] + find_exts("build_extra", ".gn", ".gni"):
+for fn in ["BUILD.gn", ".gn"] + find_exts("build_extra", [".gn", ".gni"]):
     run(["third_party/depot_tools/gn", "format", fn], env=google_env())
 
-# We use `glob()` instead of `find_exts()` in the tools directory, because:
-#   * On Windows, `os.walk()` (called by `find_exts()`) follows symlinks.
-#   * The tools directory contains a symlink 'clang', pointing at the directory
-#     'third_party/v8/tools/clang', which contains many .py files.
-#   * These third party python files shouldn't be formatted.
-#   * The tools directory has no subdirectories, so `glob()` is sufficient.
```

Comment:
Move this comment to `find_exts` definition.

Reply:
✔️